maps/Town Tutorial/Town Tutorial.eng.libx

```diff
@@ -11,20 +11,20 @@
-22:[$EAEAEA]Nice, your town now generates new [$2BEDFF]Blank Units[$EAEAEA].|You can now focus on your first resources, let's start with [$2BEDFF]Stone[$EAEAEA].[]
-23:[$EAEAEA]Ah great, you now receive your first bits of [$2BEDFF]Stone[$EAEAEA].|Yet now you still need [$2BEDFF]Wood[$EAEAEA].||After building a [$2BEDFF]Woodcutter[$EAEAEA] you will be able to build [$2BEDFF]Sawmills[$EAEAEA], to process raw [$2BEDFF]Logs[$EAEAEA] into [$2BEDFF]Planks[$EAEAEA].[]
-24:[$EAEAEA]Now that you also have your [$2BEDFF]Wood[$EAEAEA] resource chain set-up, let's start to look at [$2BEDFF]food production[$EAEAEA].||After a while your citizens will get hungry and demand food.|To prevent your citizens from starving you should provide places for your citizens to eat.[]
-25:[$EAEAEA]Great, your servants are now able to eat when they need to.|You only have a limited food supply right now, you should start producing food for your servants.|There are many ways to produce food, but we will stick to the basics, [$2BEDFF]Fish[$EAEAEA] and [$2BEDFF]Cider[$EAEAEA].[]
-26:[$EAEAEA]Sublime!|Our [$2BEDFF]food production[$EAEAEA] seems to be in a good shape for our current village.|These are the fundamental requirements to get a basic economy running within your village.|For other products you will need a [$2BEDFF]Store[$EAEAEA].[]
+21:[$EAEAEA]Amazing, you have built the [$2BEDFF]School[$EAEAEA]!|To be able to train new units you must have [$2BEDFF]Blank Units[$EAEAEA].|[$2BEDFF]Blank Units[$EAEAEA] do not perform any tasks until you assign them to a profession in the [$2BEDFF]School[$EAEAEA].|They do, however, require food.||To generate new [$2BEDFF]Blank Units[$EAEAEA] you need to build [$2BEDFF]Cottages[$EAEAEA] which will increase your allowance of citizens.[]
+22:[$EAEAEA]Excellent, your town now generates new [$2BEDFF]Blank Units[$EAEAEA]!|You can now focus on your first resources. Let's start with [$2BEDFF]Stone[$EAEAEA].[]
+23:[$EAEAEA]Great work! You will now receive your first blocks of [$2BEDFF]Stone[$EAEAEA].|However, you still require [$2BEDFF]Wood[$EAEAEA].||After building a [$2BEDFF]Woodcutter[$EAEAEA] you will be able to build [$2BEDFF]Sawmills[$EAEAEA], to process raw [$2BEDFF]Logs[$EAEAEA] into [$2BEDFF]Planks[$EAEAEA].[]
+24:[$EAEAEA]Now that you also have your [$2BEDFF]Wood[$EAEAEA] resource chain set up, let's start to look at [$2BEDFF]food production[$EAEAEA].||After a while your citizens will get hungry and demand food.|To prevent your citizens from starving you should provide places for your citizens to eat.[]
+25:[$EAEAEA]Superb, now your servants can eat when they need to.|You only have a limited food supply right now, so you should start producing it.|There are many ways to produce food, but we will stick to the basics: [$2BEDFF]Fish[$EAEAEA] and [$2BEDFF]Cider[$EAEAEA].[]
```

Comment:
I tend to fully write out what the "it" is, since people will be people and there is always the possibility that they will misinterpret the "it". But looks fine for me